boot/boot.js

```diff
@@ -1808,7 +1808,10 @@ $tw.loadWikiTiddlers = function(wikiPath,options) {
 	if(config["retain-original-tiddler-path"]) {
 		var output = {};
 		for(var title in $tw.boot.files) {
-			output[title] = path.relative(resolvedWikiPath,$tw.boot.files[title].filepath).replace(/\\/g,'/');
+			output[title] = path.relative(resolvedWikiPath,$tw.boot.files[title].filepath);
+			if(path.sep !== "/") {
+				output[title] = output[title].split(path.sep).join("/");
```

Comment:
```
output.split(path.sep).join(path.posix.sep)
```
would be the right thing here. ... But I don't think, that it actually solves the problem, that I have in mind, with windows "network paths" ... I'll need to investigate a bit more.

Reply:
Also `if (path.sep !== path.posix.sep)`.

Reply:
But what do we gain from this? Are we expecting posix path separators to change? It's just, we're catering for something that's never gonna happen.

Reply:
Anyway, I updated the PR now. But please, for gods sake, stop complaining that it now uses a ternary.